test/evergreen.yml

```diff
@@ -4196,14 +4196,12 @@ buildvariants:
   - macos-1014
   batchtime: 120 # 2 hours
   expansions:
-    posix_configure_flags:
-      -DCMAKE_C_FLAGS="-ggdb"
-      -DHAVE_DIAGNOSTIC=1
-      -DENABLE_PYTHON=1
-      -DENABLE_ZLIB=1
-      -DENABLE_STRICT=1 
-      -DCMAKE_INSTALL_PREFIX=$(pwd)/LOCAL_INSTALL
-    python_binary: 'python3'
+    # The cmake toolchain file is set to the mongodb toolchain gcc by default.
```

Comment:
Do you need to backport `posix_configure_flags`?

Reply:
I presume this was taken out in the cherry-pick because develop branch does not have it. I can put this back in if its not part of the backport.

Reply:
addressed here: [066d156](https://github.com/wiredtiger/wiredtiger/pull/9255/commits/066d156337129ca6dc484def4a8fb02160f10f27)

Reply:
Thanks for fixing